README.md

```diff
@@ -26,21 +26,37 @@ Note: docker registry must be [v2](https://docs.docker.com/registry/spec/api/).
 
 * `insecure_registries`: *Optional.* An array of CIDRs or `host:port` addresses
   to whitelist for insecure access (either `http` or unverified `https`).
-  This option overrides any entries in `ca_certs` with the same address.
+  This option overrides any entries in `certificates` with the same address.
 
 * `registry_mirror`: *Optional.* A URL pointing to a docker registry mirror service.
 
-* `ca_certs`: *Optional.* An array of objects with the following format:
+* `certificates`: *Optional.* An array of objects with the following format:
```

Comment:
Hmm, I think this is less descriptive. Does it make sense to group CA certs and client certs together? They feel like they're for different things (client verifying the server vs server verifying the client) and should be configured separately.